docs/pages/contributing/documentation/terms-usage.mdx

```diff
@@ -0,0 +1,171 @@
+| menu | Use bold for the menu name and menu item names if the user interacts with them.
+
+### Verb usage
+
+Use the following guidelines for common verbs.
+
+| Verb | How to use it
+| ---- | -------------
+| can, may, might	| Use the verb can when describing ability, capability, or capacity. Avoid using the verb may because it implies permission. Use the past tense might when describing the possibility or eventuality of an outcome.
+| clear	| Use clear rather than deselect or unselect if you have to describe removing a selection from a checkbox.
+| displays	| Use the transitive verb displays rather than the intransitive verb appears. Use displays with a direct object. For example, The command displays log messages.
+| ensure	| Use ensure to mean to make sure or to guarantee. Remember that this is not interchangeable with assure (to make confident) and insure (to provide insurance).
+| enter, type	| Use enter to instruct the user to input a value by pressing the Enter or Return key. Use type to instruct the user to type a value in a field.
+| select	| Use select to describe taking action on a menu item, checkbox, or radio button. Note that click and select are not interchangeable.
+| set up, setup	| Use set up—two words, no hyphen—when used as a verb. Don't hyphenate. Use setup—one word, no hyphen—when used as an adjective or as a noun.
+| want, wish	| Use want instead of wish or desire when the user has a choice of actions.
+
+### Word choice
+
+| Word in question | How to use it
+| ---------------- | -------------
+| affect, effect	| Use affect as a verb and use effect as a noun.
+| allow, enable	| Avoid using software as a point of view and consider rewriting to focus on the human interacting with the software.
+| app, application	| Use application or applications unless there's a specific reason for using the shorthand term app or apps.
+| back-end, front-end	| Using the hyphen in these terms is still more common than not using it. Both forms are acceptable, but for consistency use the hyphen.
+| email	| It hasn't been e-mail for thirty-plus years. Never use emails. Don't use email as a verb.
+| file name	| Use file name as two words, not filename.
+| its, it's	| Use its as a possessive meaning belonging to or associated with an object or idea previously mentioned. Because it is a vague pronoun, be sure to check that what it refers to can be easily identified. Use it's only as a contraction for it is or it has.
+| multifactor	| Use multifactor without the hyphen for multifactor authentication.
+| please	| Avoid using please in documentation unless there's a specific reason for using it. For example, you might use please if quoting the content of a message that asks the user to do something inconvenient.
+| prerequisite | As a section title, it's a bit stuffy and old-fashioned. Use Before you begin instead. If you use prerequisite in the text, there's no hyphen.
```

Comment:
Do we have a plan to change our how-to guides? They basically all have a "Prerequisites" section.